modules_core/app.js

```diff
@@ -1,12 +1,18 @@
 var plugs = require('../plugs')
 var h = require('hyperscript')
+var insertCss = require('insert-css')
 
 module.exports = {
-  needs: {screen_view: 'first'},
+  needs: {
+    screen_view: 'first',
+    styles: 'first'
+  },
   gives: 'app',
   create: function (api) {
     return function () {
-      document.head.appendChild(h('style', require('../style.css.json')))
+      process.nextTick(function () {
```

Comment:
what do you mean? Matt also used `insert-css`, the `process.nextTick` is because of how depject is currently written. see https://github.com/dominictarr/depject/pull/2 for a pull request that would remove the need for `process.nextTick`.